static/images/tardis-banner-2.svg

```diff
@@ -79,7 +79,7 @@
                id="text78"
                style="fill:#fff;fill-opacity:1">
               <tspan
-                 x="0 64.452003 151.104 237.756 324.40799 357.78"
+                 x="0 64.452003 151.104 237.756 315.40799 357.78"
```

Comment:
It causes banner to look odd, try undoing it - the space was off between I and S when you were using banner with serif fonts (that you posted in slack). For sans-serif fonts it's fine as it is - see: https://tardis-sn.github.io/tardis/_images/tardis_banner.svg